src/Event.h

```diff
@@ -22,20 +24,19 @@ struct Event {
         ValueType value;
     };
 
-
-    std::unordered_map<std::string, EventValue> eventValues;
-    //Due to strings having non-trivial destructors, I can't put them in the union! Oops!
-    std::unordered_map<std::string, std::string> stringValues;
-
     void add_value(std::string name, int arg);
     void add_value(std::string name, float arg);
     void add_value(std::string name, std::string&& arg);
 
     template<typename T>
-    T getValue(const std::string& name, T otherwise);
+    T getValue(const std::string& name, T otherwise) const;
 
 private:
 
+	std::unordered_map<std::string, EventValue> eventValues;
+	//Due to strings having non-trivial destructors, I can't put them in the union! Oops!
+	std::unordered_map<std::string, std::string> stringValues;
```

Comment:
Everything is in spaces, you should set your editor to use spaces instead of tabs.